etc/keystrokes.cfg

```diff
@@ -8,10 +8,6 @@
 <keystroke action="defer" Control="1" AltMeta="1" key="e" />
 <keystroke action="ignore" Control="1" AltMeta="1" key="BackSpace" />
 <keystroke action="force_synchronization" Control="1" AltMeta="1" key="j" />
-<keystroke action="debug_tree" Control="1" AltMeta="1" key="q" />
-<keystroke action="regions_on_screen" Control="1" AltMeta="1" key="a" />
-<keystroke action="test_input" Control="1" AltMeta="1" key="x" />
-<keystroke action="deactivate_input_devices_grab" Control="1" AltMeta="1" key="y" />
```

Comment:
How about commenting them out rather? Or rather, are those options (and the lack of being supported in normal builds) accordingly documented in the README.Keystrokes file?